src/main/java/frc/robot/subsystems/intake/RunLoader.java

```diff
@@ -1,29 +1,36 @@
 package frc.robot.subsystems.intake;
 
 import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.dashboard.Dashboard;
 
 public class RunLoader extends CommandBase{
     @SuppressWarnings({"PMD.UnusedPrivateField", "PMD.SingularField"})
   private LoaderSub loader;
-  private double power;
+  private Dashboard dashboard;
+  private boolean reversed;
+  private double coeff;
   
   /**
    * 
    * @param loader
    * @param power
    */
-  public RunLoader(LoaderSub loader, double power) {
+  public RunLoader(LoaderSub loader, Dashboard dashboard, boolean reversed) {
     this.loader = loader;
-    this.power = power;
+    this.dashboard = dashboard;
+    this.reversed = reversed;
+  
   }
 
   @Override
   public void initialize() {
+    if(reversed){coeff = -1;}
+    else{coeff = 1;}
   }
 
   @Override
   public void execute() {
-    loader.setPower(power);
+    loader.setPower(coeff*dashboard.getLoaderCoeff());
```

Comment:
I don't like this naming convention. Why is coeff for reversing the power and coeff is the dashboard power constant thingy?

Reply:
Changed it to negativeSign :)